### DISCUSSION TITLE
Bug in collaborator permissions

### DISCUSSION THREAD

**Discussion by ConeroExotico** [community/community#36818] [New to GitHub]
Good morning dear, I'm new to Git and GitHub, I have a little problem I was practicing collaborative work.

**QUESTION:**
Why can I still push and pull from a collaborator account that no longer has access according to it?

**CASE:**
- I have my personal account **_(Account-1)_** which I work within Windows and with SSH keys.
![2022-10-20_12h02_57](https://user-images.githubusercontent.com/51341851/197013317-008d983f-474d-4653-9c83-814d0b9280bd.png)

- And I created one more GitHub account for tests _**(Account-2)**_ which I work from WSL Ubuntun 20.04 and with HTTPS connection.
![2022-10-20_12h04_38](https://user-images.githubusercontent.com/51341851/197013357-9de235c4-0932-42d3-a67a-bdaad438d3b6.png)

- I have removed **_(Account-2)_** as a contributor to a test repository.

  **_View from (Account-1)_**
  ![2022-10-20_12h06_19](https://user-images.githubusercontent.com/51341851/197013635-e482d89e-3a3c-4903-9ad6-9878bcb12ff4.png)
  
  **_View from (Account-2)_**
  ![2022-10-20_12h19_55](https://user-images.githubusercontent.com/51341851/197017004-419d1414-9910-4791-a841-a59d7a914f43.png)

- Still **_(Account-2)_** has the options to push and pull the repository all within the Visual Code console.

  **_PULL_** From **_(Account-2)_** Console Visual Code

  ![2022-10-20_12h12_40](https://user-images.githubusercontent.com/51341851/197014828-fac44f97-9b42-4763-8a3d-c0c8bd73551a.png)
  
  **_PUSH_** From **_(Account-2)_** Console Visual Code

  ![2022-10-20_12h15_13](https://user-images.githubusercontent.com/51341851/197015348-b6670e1b-67f2-4e08-8d65-d6257c87142b.png)
  
  ![2022-10-20_12h21_42](https://user-images.githubusercontent.com/51341851/197016897-94126594-d4ac-4031-9c95-3f31b2e0eca1.png)

  > But within the same Ubuntu console if it asks me for a password.
  ![2022-10-20_12h29_34](https://user-images.githubusercontent.com/51341851/197018195-6447cc52-d84a-4d6f-a65f-2e7fc4ef5436.png)

**Generalities:**
- This is a private repository
- Both accounts are working within the same PC.
- **_(Account-1)_** Windows environment and with SSH keys.
- **_(Account-2)_** Ubuntu 20.04 WSL environment and with HTTPS
- I have already restarted the PC, thinking it was a cache error

**Comment by ossanna16** [comment]
Thank you for asking a good question @MrCornelius, I'm glad you found our community 🙂
